### ISSUE TITLE
Add some info on git remote control

### ISSUE THREAD

**Issue by llucenic** [MarkLodato/visual-git-guide#27]
I would really appreciate if you could add a word on remote control commands, such as pull, fetch, push etc.
You're tutorial is really great, but it is kind of unfinished regarding remote repo sync.
Thank you.

**Comment by memowe** [comment]
👍

**Comment by memowe** [comment]
I agree this would be a great idea. I don't feel competent enough to put it together alone, but if someone helped me I'd definitely invest some time to get this done.
